### ISSUE TITLE
Hero doesn't pick up gold from treasures

### ISSUE THREAD

**Issue by Silcet** [facundoolano/rpg-cli#70]
I have introduced this game at work, and a colleague and me have noticed that whenever "rpg ls" finds a treasure that contains gold the gold count of the hero doesn't increase. I attach the output from the terminal:
```fish
cmg in ❯ ~ ❯ rpg b
    potion[1]   200g
    remedy      400g
    escape      1000g

    funds: 797g
cmg in ❯ ~ ❯ rpg cd Downloads/
    wolf[1][xxxx]@~/Downloads
    wolf[1][xxxx] -30hp
    hero[3][xxxx] +10xp +59g
    hero[3][xxxx][xxx-]@~/Downloads
cmg in ❯ ~/Downloads ❯ rpg ls
cmg in ❯ ~/Downloads ❯ rpg cd Versions/
     rat[3][xxxx]@~/Downloads/Versions
    hero[3][xxxx] -2hp
     rat[3][xxxx] -33hp
    hero[3][xxxx] +33xp +38g
    hero[3][xxxx][xxxx]@~/Downloads/Versions
cmg in ❯ ~/Downloads/Versions ❯ rpg ls
📦   +405g
cmg in ❯ ~/Downloads/Versions ❯ rpg
    hero[3]@~/Downloads/Versions
    hp:[xxxxxxxxxx] 41/43
    xp:[xxxxxxxxx-] 125/155
    att:27   def:8   spd:13
    equip:{sword[1],shield[1]}
    item:{potionx7}
    894g
```

**Comment by Silcet** [comment]
I'm actually learning Rust and looking for projects to work on, so I'll take a look at the issue myself :)

**Comment by facundoolano** [comment]
Thanks for reporting! I saw this a couple of days ago. I’m rewriting that
portion of the code at the moment so it should get fixed in the next week
or so but feel free to send a PR if you want.

**Comment by Silcet** [comment]
Fix in #71
